addons/l10n_ke_edi_tremol/views/account_move_view.xml

```diff
@@ -50,6 +50,10 @@
                 <xpath expr="//field[@name='journal_id']" position="after">
                     <field name="l10n_ke_cu_invoice_number" string="Kenya CU Invoice Number" filter_domain="[('l10n_ke_cu_invoice_number', 'ilike', self')]" />
                 </xpath>
+                <xpath expr="//filter[@name='cancel']" position="after">
+                    <separator/>
+                    <filter name="l10n_ke_edi_to_send" string="To Send to TIMS" domain="[('l10n_ke_cu_invoice_number','=',False), ('state','=','posted'), ('move_type', 'in', ['out_invoice', 'out_refund']), ('country_code', '=', 'KE')]"/>
```

Comment:
country_code is a non-stored field...  (not sure if this is possible (does country_id.code work instead?) @dbkosky

Reply:
I don't know, in testing `country_code` works, but `country_id.code` doesn't

Reply:
I tested as well and it works...